lib/dradis/plugins/burp/formats/html.rb

```diff
@@ -0,0 +1,124 @@
+module Dradis::Plugins::Burp::Formats
+  # This module knows how to parse Burp HTML format.
+  module Html
+    private
+
+    def import_html(file_content)
+      logger.info { 'Parsing Burp Scanner HTML output file...' }
+      @doc = Nokogiri::HTML(file_content)
+      logger.info { 'Done.' }
+
+      # Issue headers are like: <span class="BODH0" id="X">
+      issue_headers = @doc.xpath("//span[contains(@class, 'BODH0')]")
+
+      if issue_headers.count.zero?
+        error = "Document doesn't seem to be in the Burp Scanner HTML format."
+        logger.fatal { error }
+        content_service.create_note text: error
+        return false
+      end
+
+      issue_headers.each do |header|
+        issue_id = header.attr('id')
+        html     = extract_html_fragment_for(issue_id)
+        process_html_issue(html)
+      end
+
+      logger.info { 'Burp Scanner results successfully imported' }
+      true
+    end
+
+    def process_html_issue(html_issue)
+      link       = html_issue.first.css('a').attr('href').value
+      burp_id    = link[/\/(\d+)_.*/, 1]
+      issue_id   = html_issue.attr('id').value
+      issue_text =
+        template_service.process_template(
+          template: 'issue',
+          data: html_issue
+        )
+
+      logger.info { "Processing issue #{issue_id}" }
```

Comment:
Is it possible to use the issue title here? I just feel like the logs look arbitrary:
```
...
[12:35:39] Processing evidence 12.1

[12:35:41] Processing evidence 12.2

[12:36:27] Processing issue 13

[12:36:27] Processing evidence 13
...
```